docs/application/dotnet/tutorials/certificates/managing-certificate-profile.md

```diff
@@ -46,14 +46,14 @@ Follow the same steps to create a certificate profile as described in `4. Create
 
 3. Create new Distributor for certificate profile
 
-    Follow the same steps as mentioned in `1. Create new or select existing distributor certiicate` of [Create a New Distributor Certificate](creating-certificates.md#create-a-new-distributor-certificate).
+    Follow the same steps as mentioned in [1. Create new or select existing distributor certiicate](creating-certificates.md#create_new_or_select_existing_distributor_certificate).
```

Comment:
Follow the same steps as mentioned in [create new or select existing distributor certificate](creating-certificates.md#create_new_or_select_existing_distributor_certificate).

Reply:
done